src/org/jitsi/service/neomedia/RawPacket.java

```diff
@@ -1519,6 +1519,39 @@ public void setOriginalSequenceNumber(int sequenceNumber)
     }
 
     /**
+     * Sets the padding length for this RTP packet.
+     *
+     * @param len the padding length.
+     * @return the number of bytes that were written, or -1 in case of an error.
+     */
+    public int setPaddingSize(int len)
```

Comment:
is the value of bytes written actually used?  since we only return `-1` or `2` i'm wondering if a boolean indicating success or failure would be better?